darts/timeseries.py

```diff
@@ -669,6 +669,39 @@ def shift(self, n: int) -> 'TimeSeries':
         new_series.index = new_time_index
         return TimeSeries(new_series, self.freq_str())
 
+    def diff(self, n: Optional[int] = 1,
+                periods: Optional[int] = 1,
+                dropna: Optional[bool] = True) -> 'TimeSeries':
+        """
+        Returns a differenced time series with the goal to make a time series stationary.
```

Comment:
I would slightly rephrase: `Returns a differenced time series. This is often used to make a time series stationary.`

Reply:
And done